### PR TITLE
[16.0][MIG] website_sale_checkout_skip_payment: Migration to 16.0

### PR DESCRIPTION
**Pull request by kisuka**
Migration of website_sale_checkout_skip_payment to 16.0

This PR supersedes #802

### CODE REVIEW

**Comment by kisuka** [comment]
are the codecov tests something I need to fix or is that failing due to something else unrelated?

**Comment by pedrobaeza** [comment]
Not a requirement, don't worry

File: website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js
Comment (chienandalu) [inline website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: If this tests isn't being used anymore just drop the file

Comment (kisuka) [reply website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: It's the test to ensure that the UI works correctly, shouldn't it stay? It's been present in the previous versions of the addon.

Comment (chienandalu) [reply website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: As I infer from your manifest assets it isn't declared

Comment (kisuka) [reply website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: > As I infer from your manifest assets it isn't declared

It is in the manifest

https://github.com/OCA/e-commerce/pull/803/files#diff-601ded9f4790f0d925285e921e8e4b31aed05ee1f64db08571083ddf391a851bR24

It's declared similarly to how tour tests are declared in Odoo base:
https://github.com/odoo/odoo/blob/16.0/addons/website_sale/__manifest__.py#L100

Comment (chienandalu) [reply website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: But this file is not in that route. That's what I meant

Comment (kisuka) [reply website_sale_checkout_skip_payment/static/src/js/website_sale_checkout_skip_payment_tour.js:1]: > But this file is not in that route. That's what I meant

Ah I see what you mean, I forgot to remove the old test.
Went ahead and fixed that 👍

**Comment by kisuka** [comment]
@chienandalu Fixed. Removed the old test file. Only one should be the one under the tour directory. Lemme know if it needs anything else.

**Comment by kisuka** [comment]
Does this need anymore changes to be merged?

**Review by chienandalu — APPROVED** [review]
Tested :+1: 

Could you squash your last commit into the main migration one?

please review @pilarvargas-tecnativa

**Review by pilarvargas-tecnativa — APPROVED** [review]
LGTM :) thanks!

**Pull request merged**